docs/specs/config.yml

```diff
@@ -508,11 +508,11 @@ commands:
 inputs:
   docfx.yml: |
     contribution:
-      repository: github.com/docfx/docfx
+      repository: dotnet/docfx
   docs/a.md:
 outputs:
   docs/a.json: |
-    { "content_git_url": "*github.com/docfx/docfx.zh-cn*"}
+    { "content_git_url": "*dotnet/docfx.zh-cn*"}
```

Comment:
can we remove the wildcard to make the test result more explicit? we only use wildcard only if there are no other choice.

Reply:
we do not have other choice here, the content_git_url for spec test contains rooted path which is different for different machine and platform

---
In reply to: [223995143](https://github.com/dotnet/docfx/pull/3482#discussion_r223995143) [](ancestors = 223995143)